src/Maestro/maestro-angular/src/app/page/channel-graph/channel-graph.component.html

```diff
@@ -0,0 +1,5 @@
+<div style="text-align: center;">
+    <svg stroke="currentColor" class="flowgraph" width=95% height=95% viewBox="0 0 1200 1200" preserveAspectRatio="xMinYMid meet"></svg>
```

Comment:
>0 0 1200 1200 [](start = 79, length = 13)

Why... 1200?  That's sort of weird number. Why not 1000x1000?

Reply:
I think that's not actually necessary anymore. We actually set the viewport using the bounding box in the ts.